superset/assets/javascripts/explore/stores/controls.jsx

```diff
@@ -65,7 +65,17 @@ export const controls = {
     }),
     description: 'One or many metrics to display',
   },
-
+  y_axis_bounds: {
+    type: 'BoundsControl',
+    label: 'Y Axis Bounds',
+    default: [null, null],
+    description: (
+      'Bounds for the Y axis. When left empty, the bounds are ' +
+      'dynamically defined based on the min/max of the data. Note that ' +
+      "this feature will only expand the axis range. It won't " +
```

Comment:
looks like you are using `"` since you have a `'` in the string. would be more consistent to use `"` for this whole string.